### ISSUE TITLE
Suggestion: Add searchable text & linkable headings

### ISSUE THREAD

**Issue by spen** [TheLarkInn/artsy-webpack-tour#5]
Hi @TheLarkInn!
I'm enjoying the repo - great to see an explanation of the inner workings of Webpack!

I'm wondering if there's any value in transcribing the annotations to text below each image?
It could just add a lot of extra noise but would make the readme searchable (using find etc.)

Along the same lines - a heading or some sort of quick, linkable reference could be helpful for sharing direct links to particular sections or images.

I just wanted to throw these ideas out to you, I'm not sure much they would help or hinder other folks reading the readme :)

**Comment by TheLarkInn** [comment]
I would humbly and gratefully accept PR'S in whatever format you feel best for a feature like this.

**Comment by Troesler95** [comment]
Hi guys! Love this repo, it houses a lot of great information for the flow of Webpack.

It is a bit hard to follow in my opinion though, so I was thinking of reformatting it such that the annotated text and headers are with the images that they correlate to. 

For example:

---
### [WebpackOptionsApply.js (List of plugins)](#Step1) 
<a name="Step1">![image](https://user-images.githubusercontent.com/21373376/34907934-cf6a4c34-f854-11e7-930c-c715ec1c66b9.png)</a>

"All plugins for options cases"  
"These handle any module format"

---

for each of the images and annotations.

I also think that a table of contents would help readers who might be using this to reference specific files utilize this repo more effectively rather than trying to remember where that image is in the list.

Thoughts?

**Comment by TheLarkInn** [comment]
I'm more then happy to accept any PRs for this that make it easier.
